src/esp/sensor/VisualSensor.h

```diff
@@ -108,6 +110,14 @@ class VisualSensor : public Sensor {
     return false;
   }
 
+  /**
+   * @brief Returns RenderCamera
+   */
+  virtual gfx::RenderCamera* getRenderCamera() {
```

Comment:
This should either be pure virtual or it should have a hard assert in it.  Currently this will just segfault later when you dereference the null pointer, which makes debugging harder.

Reply:
Thanks for the feedback!